data/free worlds war jobs.txt

```diff
@@ -0,0 +1,722 @@
+		government "Republic"
+		distance 2 5
+	on complete
+		payment 1000 180
+		dialog `You deliver the care package of <commodity> to the specified address and explain to the people where the care package came from. They thank you for delivering it, and you collect your payment of <payment>.`
+
+mission "FW Care Package to Republic [1]"
+	job
+	repeat
+	name `Care package to <planet>`
+	description `A citizen of the Free Worlds is worried about family members in the Republic. Immediately deliver a care package of <cargo> to <destination> by <date>. Payment is <payment>.`
+	deadline 0 1
+	cargo random 1 3
+	to offer
+		random < 30
+		has "chosen sides"
+		not "event: fw armistice"
+	source
+		government "Free Worlds"
+	destination
+		government "Republic"
+		distance 2 5
+	on complete
+		payment 1000 180
+		dialog `You deliver the care package of <commodity> to the specified address and explain to the people where the care package came from. They thank you for delivering it, and you collect your payment of <payment>.`
+
+mission "FW Care Package from Republic [0]"
+	job
+	repeat
+	name `Care package to <planet>`
+	description `A citizen of the Republic is worried about family members in the Free Worlds. Immediately deliver a care package of <cargo> to <destination> by <date>. Payment is <payment>.`
+	deadline 0 1
+	cargo random 1 3
+	to offer
+		random < 30
+		has "chosen sides"
+		not "event: fw armistice"
+	source
+		government "Republic"
+	destination
+		government "Free Worlds"
+		distance 2 5
+	on complete
+		payment 1000 180
+		dialog `You deliver the care package of <commodity> to the specified address and explain to the people where the care package came from. They thank you for delivering it, and you collect your payment of <payment>.`
+
+mission "FW Care Package from Republic [1]"
+	job
+	repeat
+	name `Care package to <planet>`
+	description `A citizen of the Republic is worried about family members in the Free Worlds. Immediately deliver a care package of <cargo> to <destination> by <date>. Payment is <payment>.`
+	deadline 0 1
+	cargo random 1 3
+	to offer
+		random < 30
+		has "chosen sides"
+		not "event: fw armistice"
+	source
+		government "Republic"
+	destination
+		government "Free Worlds"
+		distance 2 5
+	on complete
+		payment 1000 180
+		dialog `You deliver the care package of <commodity> to the specified address and explain to the people where the care package came from. They thank you for delivering it, and you collect your payment of <payment>.`
+
+
+
+mission "FW Refugees [0]"
+	job
+	repeat
+	name `Refugees to <planet>`
+	description `These <bunks> passengers would like to escape the frontlines of the war and immediately settle on <destination> by <date>. Payment is <payment>.`
+	deadline 1 1
+	passengers 2 10 .9
+	to offer
+		random < 60
+		has "chosen sides"
+		not "event: fw armistice"
+	source
+		government "Free Worlds"
+		attributes "frontline"
+	destination
+		distance 2 10
+		government "Free Worlds"
```

Comment:
I imagine there are refugees who would prefer going to Republic worlds too

Reply:
Republic space is hostile to the player most of the time, so I tried to avoid making many missions aside from the care package jobs that have a destination in Republic space. I also wouldn't expect people on FW worlds to walk up to an FW captain and ask to go to the Republic.  From https://github.com/endless-sky/endless-sky/pull/1928#issuecomment-269916809, the job board is public (I actually removed some jobs after thinking about this), so I wouldn't expect anyone to publicly state "hey, we hate the Free Worlds and we're trying to defect to the Republic in the middle of a war" by asking for transportation to Republic worlds.

Reply:
Works for me.
Perhaps then, you add a  repeating spaceport mission, conditioned such that either
 - the player has a "reasonable" reputation with the Republic that indicates they aren't doing much attacking of Republic fleets
 - the player has an absurdly high Merchant rep (not how to condition this to reflect goodwill to merchants more than it would convey hostility toward Pirates and Korath though)
 - the player has a high rep with the Syndicate, and it's people fleeing to family in Syndicate space

Reply:
The Syndicate might be its own entity, but it's still technically "the Republic." Even if the Syndicate and FW have some sort of truce between each other, I'm sure that the Navy still has jurisdiction in Syndicate space. They just don't patrol the Core since Syndicated Security does, just as they don't patrol the Deep since Deep Security does.